android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -142,6 +142,11 @@ public void updateUser(ReadableMap dataFields, Boolean mergeNestedObjects) {
 
     @ReactMethod
     public void getUserId(Promise promise) {
+        String userId = RNIterableInternal.getEmail();
```

Comment:
Doesn't the same apply to `getUserId` as `getEmail`?

Reply:
Ah.. yes totally skipped it 😬.
Thanks Victor